app/views/user/show/_show_requests.html.erb

```diff
@@ -95,6 +95,25 @@
     <% end %>
   <% end %>
 
+  <% if @private_requests.any? %>
```

Comment:
> TODO: Check that the `request_path` helper is okay to use in this
context.

Yes, the link on the page is `/request/example_title_80_2`, which redirects to `http://10.10.10.30:3000/alaveteli_pro/info_requests/example_title_80_2` when clicked. This might change depending on how we allow access to embargoed requests when a user no longer has a pro role, but works for now. Will reword on merge